docker/hail-ubuntu/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN chmod 755 /bin/retry && \
          >> /etc/apt/sources.list && \
     echo 'deb-src [signed-by=/usr/share/keyrings/deadsnakes-ppa-archive-keyring.gpg] http://ppa.launchpad.net/deadsnakes/ppa/ubuntu focal main' \
          >> /etc/apt/sources.list && \
-    hail-apt-get-install python3.7-minimal python3.7-dev python3-pip && \
+    hail-apt-get-install python3.7-minimal python3.7-dev python3-pip python3.7-distutils && \
```

Comment:
just curious. Why was this change necessary?

Reply:
Interesting that this is showing up in GitHub, this `distutils` fix is already in main and unrelated, I just gave Vedant the patch so that my PR wouldn't hold up development on this branch.